rales_engine/app/models/item.rb

```diff
@@ -1,3 +1,9 @@
 class Item < ApplicationRecord
+  has_many :invoice_items
+  has_many :invoices, through: :invoice_items
 
+  def self.most_items(quantity)
```

Comment:
make sure you have a test for this method on the unit level to ensure that it works properly 👍